src/Vandelay.cpp

```diff
@@ -1,26 +1,45 @@
-#include "VandelayWindow.h"
+
 #include "Vandelay.h"
 
+const char * kSignature = "application/x-vnd.Blum-Vandelay";
 
-int main(int, char**)
-{	
-	VandelayApp VanApp;
 
-	VanApp.Run();
-
-	return(0);
+VandelayApp::VandelayApp() : BApplication(kSignature) {
+	
+	
```

Comment:
Extra and unnecessary spaces at beginning of function. Also please conform to Haiku coding style (https://www.haiku-os.org/development/coding-guidelines/) as much as possible re. function formatting.